queries/haskell/highlights.scm

```diff
@@ -116,7 +116,8 @@
 
 
 (module) @namespace
-(qualified_module (module) @constructor)
+((qualified_module (module) @constructor)
+ (module))
```

Comment:
what is this second `module`

Reply:
Also, as this is some form of `((a) (b))` query, it will lead to unwanted captures, i.e. `a a b b` is 4 matches when I think you probably only want it to be one?

Reply:
This is to match
```haskell
import qualified Data.Map as Map
                -- ^ @constructor
                           -- ^ @namespace
```

(inspired by how it's done in Helix).
The second `(module)` should be anchored.